TransferMethod/Sources/SelectTransferMethodTypePresenter.swift

```diff
@@ -139,9 +140,14 @@ final class SelectTransferMethodTypePresenter {
                             ]
                             strongSelf.loadSelectedCountry(countries, with: user?.country)
                             strongSelf.loadCurrency(result)
-                            strongSelf.loadTransferMethodTypes(result)
+                            strongSelf.getFeeAndProcessingTime { (result) in
```

Comment:
keep the name as whatever you decide in core sdk, like getXXX

Reply:
Renamed to `retrieveTransferMethodTypesFeesAndProcessingTimes`

Reply:
Renamed to `retrieveTransferMethodTypesFeesAndProcessingTimes`